ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/persistence/PostgresAround.scala

```diff
@@ -90,6 +90,7 @@ trait PostgresAround {
     val tempDir = Files.createTempDirectory("postgres_test")
     val tempDirPath = tempDir.toAbsolutePath.toString
     val dataDir = Paths.get(tempDirPath, "data")
+    val postgresConfPath = Paths.get(dataDir.toString, "postgresql.conf");
```

Comment:
The logic here was just completely broken. The config file was not used at all, the ports weren’t used either and instead it connected via unix domain sockets.